collectors/pod.go

```diff
@@ -137,6 +137,18 @@ var (
 		"The limit on memory to be used by a container in bytes.",
 		[]string{"namespace", "pod", "container", "node"}, nil,
 	)
+
+	descPodContainerResourceRequestsGPU = prometheus.NewDesc(
+		"kube_pod_container_resource_requests_gpu",
```

Comment:
@others if other people do not have second opinion on this, I will change to kube_pod_container_resource_limits_gpu_devices today